tests/test_project/templates/dashboard_test.html

```diff
@@ -1 +1,2 @@
 <div style="display:none">Testing dashboard</div>
+<div style="display:none">{{ test_extra_config }}</div>
```

Comment:
Let's add an if so this extra div is added only if needed.

Reply:
Hey, @nemesisdesign, this is just the test code for UTs, it's not suppose to be exposed to user and in the UTs, it will always return `true`.
So, do you think adding that would be helpful?

Reply:
yes, I think we have to, it's a question of approach: it's not good to include data which is not going to be used, if we allow this kind of practice we'll soon end up with many cases like this with the result that the response data will grow unnecessarily.
We should just stick with best practices whenever possible, unless there's a good reason to do it differently.

Reply:
Done.